requirements.txt

```diff
@@ -1,35 +1 @@
-#
-# This file is autogenerated by pip-compile
-# To update, run:
-#
-#    pip-compile --output-file=requirements.txt requirements.in
-#
--e git+https://github.com/edx/django-rest-framework.git@1ceda7c086fddffd1c440cc86856441bbf0bd9cb#egg=djangorestframework==3.6.3  # via -r requirements.in
-astroid==1.6.5            # via pylint
-backports.functools-lru-cache==1.5  # via astroid, pylint
-configparser==3.5.0       # via pylint
-coverage==4.5.1           # via -r requirements.in
-ddt==1.4.1                # via -r requirements.in
-django-crum==0.7.3        # via -r requirements.in
-django-fake-model==0.1.4  # via -r requirements.in
-django-mysql==2.4.1       # via -r requirements.in
-django==1.11.29           # via -r requirements.in, django-crum, django-mysql, jsonfield
-edx-opaque-keys==0.4.4    # via -r requirements.in
-enum34==1.1.6             # via astroid, ddt
-funcsigs==1.0.2           # via mock
-futures==3.2.0 ; python_version < "3.0"  # via -r requirements.in, isort
-isort==4.3.4              # via pylint
-jsonfield==2.0.2          # via -r requirements.in
-lazy-object-proxy==1.3.1  # via astroid
-mccabe==0.6.1             # via pylint
-mock==2.0.0               # via -r requirements.in
-path.py==8.2.1            # via -r requirements.in
-pbr==5.1.1                # via mock, stevedore
-pycodestyle==2.4.0        # via -r requirements.in
-pylint==1.9.3             # via -r requirements.in
-pymongo==3.7.2            # via edx-opaque-keys
-pytz==2018.7              # via django
-singledispatch==3.4.0.3   # via astroid, pylint
-six==1.11.0               # via astroid, edx-opaque-keys, mock, pylint, singledispatch, stevedore
-stevedore==1.30.0         # via edx-opaque-keys
-wrapt==1.10.11            # via astroid
+-r requirements/base.txt
```

Comment:
why do you keep this file ?

Reply:
should i delete it? it's used in the Makefile in make requirements

Reply:
Yes, you can delete this and change the Mafefile requirements target to use the new requirements file (base.txt)

Reply:
i removed the file